health/arg_codes.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package health
+
+// ArgCode is a type for the key to be used in the Args of a Warnable.
+type ArgCode string
+
+const (
+	// ArgCodeAvailableVersion is an arg code used in update availability Warnables to provide the available version of the Tailscale client.
+	ArgCodeAvailableVersion ArgCode = "available-version"
```

Comment:
Maybe we can drop "Code" from all these symbol names and just be `ArgAvailableVersion ArgKey = "..."`

There's no other type of "Arg" anyway, right?

Reply:
done